### PR TITLE
Feature/start admin dashboard

### PR DESCRIPTION
**Pull request by starullo**
# Description

Added the Admin Dashboard component, a component to render incidents waiting to be reviewed, and a component that displays more data if toggled. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by willwearing — APPROVED** [review]
Looks good!

File: src/index.js [hunk 1]
```diff
@@ -62,6 +63,9 @@ function App() {
         <Route path="/about">
           <About />
         </Route>
+        <Router path="/admin-dashboard">
```
Comment (victoriamount) [inline src/index.js]: I already messaged about this, but thought I'd throw in here too :) 

Router --> Route, then we're good to merge!

**Review by victoriamount — COMMENTED** [review]
Ready to merge as soon as the router change is made! Would this be a time to directly edit in github? It's such a tiny change!

File: src/index.js (hunk 1 above)
Comment (starullo) [reply src/index.js]: Changed it! I made changes to the branch that was already pushed to github. Good suggestion

**Review by victoriamount — APPROVED** [review]

**Review by mcolsen — APPROVED** [review]
Looks good! I'm comfortable merging this into main

**Pull request merged**